packages/chakra-ui-core/src/Editable/index.js

```diff
@@ -274,7 +273,7 @@ const EditableInput = {
       },
       nativeOn: {
         blur: event => {
-          submitOnBlur ? onSubmit() : onCancel()
```

Comment:
With CustomControls(you can see it in stories) when i click the save buton it causes input to blur so it always calls the `onCancel` method. `onSubmit` never gets called. So the text never updates.

Reply:
I see. Good catch. Let me take alook at the review